utils/utils.h

```diff
@@ -174,9 +174,10 @@ struct gbXdata {
   char data[];
 };
 
-struct gbCreate3 {
```

Comment:
Why changing the name now ? Not the intrest of this patch atleast. Lets keep as it is, this is making lot of diff IMO

Reply:
Because the gbCreate3 is not very correct what it means, it not only for the v3, it will be for all the possible versions followed. It will be confusing for others or us in future to read it back.

Reply:
@lxbsz 
I get it, but this don't have to be clubbed with this PR, lets do it later, the PR looks quite confusing. Its fine to leave with gbCreate3 for now.

Reply:
We can clean the code later.

Reply:
sure.